packages/next/src/build/index.ts

```diff
@@ -1487,7 +1487,8 @@ export default async function build(
           'Building'
         )
         const promises: Promise<any>[] = []
-        const sema = new Sema(10)
+        const sema = new Sema(5)
+        let extraSema = 5
```

Comment:
It looks like this is some sort of hack to avoid spawning too much work at the start of the build where lots of warmup work is happening, but then to ramp back up after we've had a chance to initialize?

This feels unintuitive. Can you put a comment here?